packages/migrate/src/utils/ensureDatabaseExists.ts

```diff
@@ -225,6 +225,7 @@ export function prettifyProvider(provider: ConnectorType): PrettyProvider | stri
   switch (provider) {
     case 'mysql':
       return `MySQL`
+    case 'postgres':
     case 'postgresql':
```

Comment:
Is this still used?

Reply:
Yes though it would be nice to get rid of it everywhere